py/server/deephaven/experimental/iceberg.py

```diff
@@ -116,47 +289,20 @@ def tables(self, namespace: str) -> Table:
             a table containing the tables in the provided namespace.
         """
 
-        if namespace is not None:
-            return Table(self.j_object.listTablesAsTable(namespace))
-        return Table(self.j_object.listTablesAsTable())
+        return Table(self.j_object.tables(namespace))
 
-    def snapshots(self, table_identifier: str) -> Table:
+    def load_table(self, table_identifier: str) -> IcebergTable:
```

Comment:
The return types of some of these methods are not matching, like `load_table` should return `IcebergTableAdapter` whereas `tables` should return `Table`.
Can you please verify once?